documents/manifesto.md

```diff
@@ -0,0 +1,28 @@
+# The Lua Users Foundation Manifesto
+
+The Lua Users Foundation (the Foundation) is an association of individuals in a modular umbrella structure consisted of several workgroups, detailed in the Lua Users Foundation Constitution.
+The Foundation formal decision-making process is governed by the Lua Users Foundation Constitution, according to the Foundation Goals and Values, towards our Mission.
+
+## The Mission
+
+  * Support and promote the Lua programming language, its community and ecosystems.
+    
+## The Values
+
+  * The Lua programming language is developed by the Lua Team and defined by the Lua Programming Manual. Furthermore:
```

Comment:
Should clarify on what the language is. Is it the language as described in the manual? Or is it the reference implementation?

Reply:
This is a difficult question, which will block the development of the manifesto. I say that one of the goals of the LUF is to make this point officially clarified by the Lua Team 

There were many discussions on that point with Roberto on previous Lua in Moscow. Please refer to: https://www.youtube.com/watch?v=wdRGOE1N-FA&list=PLTHTttj90_1mdhrKc-H5apZWoVIXmkhcN (in English), especially second and third talks.

Reply:
@Etiene's comment at https://github.com/lua-users-foundation/foundation/pull/3#discussion_r174491714 implies that we've already selected an answer to this question.
As far as I know, LuaJIT *is* compliant to the lua 5.1 manual.
Which then implies that "the lua language" in this paragraph is intended to be the reference implementation; not what is described in the manual.